whelktool/src/main/groovy/datatool/WhelkTool.groovy

```diff
@@ -687,6 +691,10 @@ class WhelkTool {
             it.flush()
             it.close()
         }
+        if (errorDetected) {
+            log "Script terminated due to an error, see $reportsDir/ERRORS.txt for more info"
+            System.exit(2)
```

Comment:
```suggestion
            System.exit(1)
```
1 is conventionally the generic "an error occurred" exit code (in bash, exit code 2 means "Misuse of shell builtins")